cypress/integration/group_page_test.js

```diff
@@ -80,5 +80,31 @@ describe('Group page tests', () => {
         cy.get('[role="dialog"]').get('button').contains('Remove').click();
       });
     });
+
+    it.only('correctly removes a device from a group with symbols in its name', function () {
+      const groupName = '!"£$%&/()=?^';
+      const encodedGroupName = encodeURIComponent(groupName);
+      const groupFixture = 'group.special-characters.devices.json';
+      cy.fixture(groupFixture).then((groupDevices) => {
+        cy.server();
+        cy.route(
+          'GET',
+          `/appengine/v1/${this.realm.name}/groups/${encodedGroupName}/devices?details=true`,
+          groupDevices,
+        );
+        cy.route({
+          method: 'DELETE',
+          url: `/appengine/v1/${this.realm.name}/groups/${encodedGroupName}/devices/*`,
+          status: 204,
+        }).as('deleteDeviceCheck');
```

Comment:
I think `deleteDeviceRequest` might be a better naming here than `deleteDeviceCheck`, following the pattern we already used in other test files.